server/context.go

```diff
@@ -318,22 +322,35 @@ func parseDurationEnv(env, internalName string, duration *time.Duration) {
 // when NewContext is called.
 func (ctx *Context) readEnvironmentVariables() {
 	// cockroach-linearizable
-	if linearizableString := os.Getenv("COCKROACH_LINEARIZABLE"); len(linearizableString) != 0 {
-		if linearizable, err := strconv.ParseBool(linearizableString); err != nil {
-			log.Errorf("could not parse environment variable COCKROACH_LINEARIZABLE=%s, setting to default of %t, error: %s",
-				linearizableString, ctx.Linearizable, err)
+	linearizableEnv := "COCKROACH_LINEARIZABLE"
+	if value := os.Getenv(linearizableEnv); len(value) != 0 {
```

Comment:
How about combining these two boolean checks into a parseBoolEnv function to be more DRY?